packages/sdk/src/mcp/workflows/api.ts

```diff
@@ -255,6 +257,10 @@ export interface WorkflowBindingImplOptions {
   resourceWorkflowRead: (
     uri: string,
   ) => Promise<{ data: z.infer<typeof WorkflowDefinitionSchema> }>;
+  resourceWorkflowUpdate: (
```

Comment:
Does it make sense to edit a run? A run should be a fact, not a configuration. Why do we need to edit a run?